src/com/sap/piper/tools/ToolKey.groovy

```diff
@@ -0,0 +1,26 @@
+package com.sap.piper.tools
+
+
+enum ToolKey {
+
+    JAVA ('JAVA_HOME', ''),
+    MTA ('MTA_JAR_LOCATION', 'mtaJarLocation'),
+    NEO ('NEO_HOME', 'neoHome'),
+    CM_CLI ('CM_CLI_HOME', 'cmCliHome')
+
+    final def environment
```

Comment:
You can define all the environment variables that you want, also MTA_JAR_LOCATION.

Reply:
I was confused, above not environment variable, but there is not xxx_HOME for the mta jar file, since this is not a tool in the same way like e.g. ```JAVA_HOME```, ```NEO_HOME```, you are right, of course anybody can set any environment variable, hence also MTA_JAR_LOCATION.